patches/tModLoader/Terraria.ModLoader/ModSound.cs

```diff
@@ -12,7 +13,7 @@ public class ModSound
 		/// <summary>
 		/// The SoundEffect instance of the sound that this ModSound controls. This is here so you can call CreateInstance on it.
 		/// </summary>
-		public SoundEffect sound
+		public SoundWrapper sound
```

Comment:
This breaks all mods that call `sound.CreateInstance();` See ExampleMod's Wooo.cs.